src/main/kotlin/demo/dto/interview/InterviewPayloads.kt

```diff
@@ -0,0 +1,7 @@
+package demo.dto.interview
+
+import java.sql.Timestamp
+import java.time.Instant
+
+data class InterviewPayloads(val dateTime: Timestamp = Timestamp.from(Instant.now()),
```

Comment:
Do czego to służy?

Reply:
Bo myślałem że może się przydać, bo używamy payloady przy innych endpointach więc stwierdziłem że warto dodać